File: src/main/java/net/guneyilmaz0/permissionss/commands/PermissionSCommands.java
package net.guneyilmaz0.permissionss.commands;

import cn.nukkit.Server;
import cn.nukkit.command.Command;
import cn.nukkit.command.CommandMap;

import java.util.ArrayList;
import java.util.List;

public class PermissionSCommands {
    public static void init() {
        CommandMap map = Server.getInstance().getCommandMap();
        List<Command> commands = new ArrayList<>();
        commands.add(new AddGroup());
        commands.add(new RemoveGroup());
        commands.add(new SetGroup());
        commands.add(new ListGroups());
        commands.add(new DefaultGroup());
        commands.add(new PermissionsSInfo());
        commands.add(new SetFormat());
        commands.add(new SetUserPermission());
        commands.add(new UnsetUserPermission());
        map.registerAll("permissionss", commands);
    }
}

File: src/main/kotlin/commands/PermissionSCommands.kt
package net.guneyilmaz0.permissions.commands

import cn.nukkit.Server
import cn.nukkit.command.Command
import cn.nukkit.command.CommandMap
import commands.*

object PermissionSCommands {
    fun init() {
        val map: CommandMap = Server.getInstance().commandMap
        val commands: MutableList<Command> = ArrayList()
        commands.add(AddGroup())
        commands.add(RemoveGroup())
        commands.add(SetGroup())
        commands.add(ListGroups())
        commands.add(DefaultGroup())
        commands.add(PermissionsSInfo())
        commands.add(SetFormat())
        commands.add(SetUserPermission())
        commands.add(UnsetUserPermission())
        map.registerAll("permissionss", commands)
    }
}